firstvoices/backend/management/commands/copy_site.py

```diff
@@ -271,17 +271,17 @@ def copy_galleries(source_site, target_site, image_map, set_modified_date, logge
             gallery.cover_image_id = image_map[gallery.cover_image.id]
         else:
             logger.warning(
-                f"Missing gallery.cover.image in image map with id: {gallery.cover_image.id}"
+                f"No cover_image or gallery.cover_image not present in image map. Gallery Id: {gallery.id}."
```

Comment:
[minor] the double negative here might be confusing, maybe change to :

"No cover_image or gallery.cover_image present in image map. Gallery Id: {gallery.id}."